src/reporters/json.ts

```diff
@@ -248,7 +247,7 @@ export async function getScreenshotBlocks(screenshot: Buffer) {
     const top = row * blockHeight;
     for (let col = 0; col < divisions; col++) {
       const left = col * blockWidth;
-      const buf = await img
+      const buf = await sharp(screenshot, { sequentialRead: true })
```

Comment:
Can we add a comment here explaining why we do this so someone doesn't come along later and optimize this away?

Reply:
Done.